frontend/src/components/auth/auth.js

```diff
@@ -0,0 +1,51 @@
+import { auth } from '../../../../backend/firebase/firebase'
+
+import {
+    signInWithEmailAndPassword as firebaseSignInWithEmailAndPassword,
+    createUserWithEmailAndPassword as firebaseCreateUserWithEmailAndPassword,
+    signOut as firebaseSignOut,
+    GoogleAuthProvider,
+    FacebookAuthProvider,
+    signInWithPopup
+} from 'firebase/auth';
+
+export const signInWithEmailAndPassword = (email, password) => {
+    return firebaseSignInWithEmailAndPassword(auth, email, password);
+};
+
+export const createUserWithEmailAndPassword = (email, password) => {
```

Comment:
Does implementing user email and password increase the complexity of your data model and permission handling? IMO google/facebook is enough @ocorrodi @jgsiqueira what are your thoughts here?

Reply:
Sign up is one of the requirements so I think that'd probably have to be with email/password? But if it adds a lot of complexity we can definitely discuss and I can check with the intern director.

Reply:
Login in with Facebook is a little more complicated than Google's. It requires a full app review from meta developers' side. Says it takes approximately 5-10 days. In terms of increasing complexity of the data model, I don't think it will make much difference. Each user whether through Google or email/ password receives a user id (token).

Reply:
Sounds good, I believe you can still implement fb login before review and the review just has to happen before release (which we prob don't need to do for now)? But that might be a bit outdated. Also don't think fb necessary if there's a blocker there

Reply:
Just my 2 cents (a bit late) here, I would recommend focus on essential features, in this case email/password.

Nice to add Gmail/Facebook, but that is a stretch feature, let's not spend too much time on those if they start to consume too much time.